src/jenkins_job_insight/ai_models.py

```
@@ -0,0 +1,313 @@
+                "Cursor CLI binary 'agent' not found — cannot list cursor models"
+            )
+            return []
+        except asyncio.TimeoutError:
+            logger.warning(
+                "Cursor 'agent models' subprocess timed out after %ds",
+                _CURSOR_SUBPROCESS_TIMEOUT,
+            )
+            try:
+                proc.kill()  # type: ignore[union-attr]
+            except Exception:
+                pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)